bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/DefaultEntryLogger.java

```diff
@@ -528,6 +528,7 @@ public boolean removeEntryLog(long entryLogId) {
         }
         if (!entryLogFile.delete()) {
```

Comment:
above we return false in case of file not found
```java
        } catch (FileNotFoundException e) {
            LOG.error("Trying to delete an entryLog file that could not be found: "
                    + entryLogId + ".log");
            return false;
        }
```
Would it make sense to simply log and return true; after all, we wanted to delete the entry log, and let the `entryLogMetaMap.remove` run.
Alternatively, throw IllegalStateException to force an investigation into how we got into such a state (is there a concurrent process that could delete the entry log?) though this feels like an overkill.

Reply:
I think the first method is better, simply log and return true, throw IllegalStateException changes the previous behavior, users may be less willing to care about the reason for its non-existence when cleaning up the outdated entryLogFile. And keep the original behavior  can avoid introducing possible bugs. Here, we just make `entryLogMetaMap.remove` work